src/vl-mapactions-custom-map.js

```diff
@@ -115,21 +129,7 @@ export class VlCustomMap extends VlMapWithActions {
   }
 
   initializeView(boundingBox, maxZoom) {
-    if (!this.getView().getZoom()) {
-      const view = new View({
-        extent: this.projection.getExtent(),
-        projection: this.projection,
-        maxZoom: 16,
-        minZoom: 2,
-        center: [140860.69299028325, 190532.7165957574],
-        zoom: 2,
-      });
-      this.zoomViewToExtent(view, boundingBox, maxZoom);
-      this.setView(view);
-      if (this.overviewMapControl) {
-        this.addControl(this.overviewMapControl); // control needs to be added after view initialization
```

Comment:
als de comment klopt, dan is er een probleem nu? 
want wordt nu niet meer daarna gedaan?

Reply:
Blijkt alleszins niet uit de testen, dus potentieel niet meer nodig door OL upgrades. Voor de zekerheid kunnen afnemers best deze PR's eerst testen voordat een merge en release uitgevoerd wordt.

Reply:
De view wordt nu bij constructie gezet, deze was niet afhankelijk van boundingbox of maxZoom, dus normaal zou dit geen problemen mogen geven. De bijkomende acties zijn behouden of werden ook al eerder uitgevoerd.